provider.tf

```diff
@@ -9,9 +6,10 @@ terraform {
     }
   }
 }
-# provider block supplies the info needed to access aws
+
 provider "aws" {
   region                   = "us-west-2"
   shared_credentials_files = ["~/.aws/credentials"]
   profile                  = "kenna"
-}
\ No newline at end of file
+}
+
```

Comment:
Looks like you have more than one newline at the end of the file.
Let's have only one.

Reply:
Deleted extra line at end of file